pkgs/by-name/ro/roslyn-ls/deps.json

```diff
@@ -29,6 +29,24 @@
     "hash": "sha256-WnXjX2pGroKn+KukCgPeUUZAkz2S6CdMrCc9P2VcBDQ=",
     "url": "https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/45bacae2-5efb-47c8-91e5-8ec20c22b4f8/nuget/v3/flat2/messagepack.annotations/2.5.198/messagepack.annotations.2.5.198.nupkg"
   },
+  {
+    "pname": "Microsoft.AspNetCore.App.Runtime.linux-arm64",
```

Comment:
These runtime deps should be coming from nixpkgs I think.

Reply:
OK, see the latest changes, this was the important change:
```
    # keeps unnecessary platform-specific packages out of deps.json
    "-p:EnableAppHostPackDownload=false"
    "-p:EnableRuntimePackDownload=false"
```